Dockerfile.sdk

```diff
@@ -167,6 +167,14 @@ RUN if [ "$TRITON_ENABLE_GPU" = "ON" ]; then \
       fi \
     fi
 
+# Install Java API Bindings
```

Comment:
Can you put this at line 136 so it is next to where we build the other client components.